projects/tinkerbell/tinkerbell-crds/chart/templates/rufio/bmc.tinkerbell.org_jobs.yaml

```diff
@@ -4,6 +4,7 @@ kind: CustomResourceDefinition
 metadata:
```

Comment:
we do we have a seperate chart for crds? does the upstream chart not include these?